### ISSUE TITLE
Add Logging

### ISSUE THREAD

**Issue by vmtyler** [vmtyler/vascodagama#4]
We need to add logging back in- don't love papertrail though. Getting crashes and need to track it down

**Comment by mcowger** [comment]
Do you have a preference on service?  Happy to make it work with whatever you want to show off...

**Comment by vmtyler** [comment]
not really. i was considering setting up an elk server on aws or vca and using that..


On April 9, 2015 at 12:02:57 PM, Matt Cowger (notifications@github.com) wrote:

Do you have a preference on service? Happy to make it work with whatever you want to show off...

**Comment by mcowger** [comment]
Up to you - if you setup an ELK stack I will make it work.

If you want to use loggly we can do that too.

Also, we can do both (its just an extra handler).

—M

**Comment by vmtyler** [comment]
I’ll setup the ELK server and let you know. Do you want to setup logging using a user-provided services from CF?


On April 9, 2015 at 12:05:05 PM, Matt Cowger (notifications@github.com) wrote:

Up to you - if you setup an ELK stack I will make it work.

If you want to use loggly we can do that too.

Also, we can do both (its just an extra handler).

—M

**Comment by mcowger** [comment]
Sure - I feel like it belongs in there with the rest of the config.
